src/borg/item.py

```diff
@@ -110,22 +119,22 @@ class Item(PropDict):
     If an Item shall be serialized, give as_dict() method output to msgpack packer.
     """
 
-    VALID_KEYS = set(key.decode() for key in ITEM_KEYS)  # we want str-typed keys
+    VALID_KEYS = ITEM_KEYS | {'deleted'}  # str-typed keys
```

Comment:
"deleted" is a key used by "borg diff" internally but not written into the repo.